lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -52,9 +52,13 @@ public function createDatabase($database)
         $params = $this->_conn->getParams();
         $driver = $params['driver'];
         $options = array(
-            'driver' => $driver,
-            'path' => $database
+            'driver' => $driver
         );
+        if(isset($params['memory']) && $params['memory']){
```

Comment:
@mikeSimonson shouldn't `createDatabase()` rather be a noop in this case? It doesn't make sense to connect afterwards and immediately close again, as in-memory database only exists during the connection, right?